src/api/uploads/PlainUpload.js

```diff
@@ -126,8 +127,10 @@ class PlainUpload extends BaseUpload {
         successCallback = noop,
         errorCallback = noop,
         progressCallback = noop,
+        conflictCallback = null,
```

Comment:
It looks like the callback has a default value in some places, but not in others. Should it be consistent?